SpatialGDK/Documentation/content/non-unreal-server-worker-types.md

```diff
@@ -82,19 +104,69 @@ type UnrealResponse {
     string some_response_string = 1;
 }
 
-component NonUnrealAuthoritative {
+component TestComponent {
     id = 1337;
     uint32 counter = 1;
+    command UnrealResponse test_command(UnrealRequest);
 }
 
-component UnrealAuthoritative {
+component OtherTestComponent {
     id = 1338;
     uint32 other_counter = 1;
-    command UnrealResponse test_command(UnrealRequest);
 }
 ```
 
-#### Send data
+#### Using the code generator example
+
+Using the [Third Person Shooter project](https://github.com/spatialos/UnrealGDKThirdPersonShooter) as an example, and the above schema file placed in the directory `./spatial/schema/improbable/`, the command below will output code to a new `./Game/Source/ThirdPersonShooter/ExternalSchemaCodegen` folder:
+
+`./Game/Plugins/UnrealGDK/SpatialGDK/Build/Scripts/ExternalSchemaCodegen.bat ./spatial/schema/my_external_schema ./Game/Source/ThirdPersonShooter/ExternalSchemaCodegen`
```

Comment:
Change to "MyProject"

Reply:
We should be using cmd style here such as `Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\ExternalSchemaCodegen.bat spatial\schema\session Game\Source\GDKShooter\ExternalSchemaCodegen`